mhs-reference-implementation/selenium_tests/page_objects/build_scr.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+
+from src.gp_summary_update import SummaryCareRecord
+from utilities.message_utilities import MessageUtilities
+
+HL7_TIMESTAMP_FORMAT = "%Y%m%d%H%M%S"
+
+
+def build_scr(asid, patient_nhs_number, human_readable_payload):
+    """Build a GP summary upload message
+
+    :param asid: The ASID of this node.
+    :param patient_nhs_number: The NHS number of the patient this record belongs to.
+    :param human_readable_payload: The himan readable payload to be included in the summary message.
+    :return: The GP summary message.
+    """
+
+    current_utc_time = datetime.datetime.utcnow()
+    hl7_timestamp = current_utc_time.strftime(HL7_TIMESTAMP_FORMAT)
+
+    gp_summary_properties = {
+        "Id": MessageUtilities.get_uuid(),
```

Comment:
Might be worth moving this dictionary to a json file in the data folder and loading it with JSON.loads (think there is  a json loading method in the utilities package). Then just update the variables we receive as parameters as needed.